Use [rel="noopener"] on external links

nscyclone opened the pull request:
`[rel="noopener"]` is added to external links having `[target="_blank"]`  to deny access to `window.opener`. [Here](https://developers.google.com/web/tools/lighthouse/audits/noopener) is a quick rundown on why it is important.

OvermindDL1 commented:
> [target="_blank"] is added to links to source code in order to open them in a new tab;

Please no, like *really* **no**.  If I want to open a new tab I will middle-click.  Left-click should always *only* open in the same tab and it is really **really** irritating when a site overrides that behaviour so I have no option to open in the same tab then...

nscyclone commented:
Thanks, @OvermindDL1! You're right, there was no sense in changing current behaviour on links to source code. I've rebased the branch and updated the PR description accordingly.

whatyouhide commented:
Thanks @nscyclone! 💟

whatyouhide commented:
@nscyclone missed you're a committer on ex_doc, sorry for the merge + usual thanks 😄
